pkg/testcase/cluster.go

```diff
@@ -24,13 +24,17 @@ func TestBuildCluster(g GinkgoTInterface) {
 		fmt.Println("Backend:", cluster.Config.ExternalDb)
 	}
 
-	if cluster.Config.ExternalDb != "" && cluster.Config.DataStore == "" {
-		for i := 0; i > len(cluster.ServerIPs); i++ {
-			cmd := "grep \"datastore-endpoint\" /etc/systemd/system/k3s.service"
-			res, err := shared.RunCommandOnNode(cmd, cluster.ServerIPs[0])
-			Expect(err).NotTo(HaveOccurred())
-			Expect(res).Should(ContainSubstring(cluster.Config.RenderedTemplate))
-		}
+	if cluster.Config.ExternalDb != "" && cluster.Config.DataStore == "external" {
```

Comment:
Dont we need to add the check for product k3s here?

Reply:
nope, bc we dont have this vars on rke2 tfvars